documentation/docs/03-loading.md

```diff
@@ -58,7 +58,7 @@ The `load` function receives an object containing six fields — `url`, `params`
 
 #### url
 
-`url` is an instance of [`URL`](https://developer.mozilla.org/en-US/docs/Web/API/URL), containing properties like the `origin`, `hostname`, `pathname` and `searchParams`.
+`url` is an instance of [`URL`](https://developer.mozilla.org/en-US/docs/Web/API/URL), containing properties like the `origin`, `hostname`, `pathname` and `searchParams` (which contains the parsed query string as a [URLSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) object).
```

Comment:
```suggestion
`url` is an instance of [`URL`](https://developer.mozilla.org/en-US/docs/Web/API/URL), containing properties like the `origin`, `hostname`, `pathname` and `searchParams` (which contains the parsed query string as a [`URLSearchParams`](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) object).
```

For consistency we should probably do this.

Reply:
oops